cypress/support/commands.js

```diff
@@ -22,4 +22,38 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const { faker } = require('@faker-js/faker');
+
+const generateWorkers = () => {
```

Comment:
commands.js isnt good file for generateWorker function, its for cypress commands. You can use cypress config file and use "task"